README.md

```diff
@@ -93,9 +93,9 @@ In **PowerShell** or in a **Command Prompt** window:
         sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64
     ```
 
-- **OS X**
+- **macOS**
 
-  Most problems on OS X have been with loading and linking OpenSSL and zlib. The following commands install necessary packages and tell the Python installer where to find them.
+  Most installation problems on macOS have involved loading and linking `OpenSSL` and `zlib`. The following command installs the necessary packages and tell the Python installer where to find them.
```

Comment:
: (colon) at end instead of period.

Reply:
AWS style guidelines call for a period here.

Reply:
Good to know. Need to be consistent. I believe then that there is one area where we use a colon that will need to be updated.